apps/shadcn-example/src/components/wallet-selector.tsx

```diff
@@ -0,0 +1,145 @@
+"use client";
+
+import {
+  AptosWallet,
+  WalletItem,
+  isInstallRequired,
+  partitionWallets,
+  truncateAddress,
+  useWallet,
+} from "@aptos-labs/wallet-adapter-react";
+import { ChevronDown, Copy, LogOut } from "lucide-react";
```

Comment:
Is this a package included in the shadcn installation?

Reply:
Yes, it's the default icon pack that's included with shadcn